website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -2,79 +2,95 @@
+<Tabs groupId="configurations">
+  <TabItem label="Minikube" value="Minikube">
 
-:::note
-By default, the ingress controller will not have data compression enabled. You can enable it by following the steps listed in the [Nginx ingress controller compression](./#nginx-ingress-controller-compression-gzip) section.
-:::
+  1. [Setup kubectl](https://minikube.sigs.k8s.io/docs/handbook/kubectl/)
+  2. Ensure that you have a default storage class running. If not, [enable addon default-storage class](https://kubernetes.io/docs/tutorials/hello-minikube/#enable-addons)
+  3. The NGINX Ingress Controller should be enabled by default. If not, [set up Ingress on Minikube with the NGINX Ingress Controller](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/).
+  Ensure that you enable the correct version for your cluster.
 
-### Installing the Chart
+</TabItem>
+<TabItem label="Google Cloud Kubernetes" value="GoogleCloudKubernetes">
 
-1. Add Appsmith into your repository using Helm.
+  1. [Configure cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) on Google Cloud Kubernetes.
+  2. Ensure that you have a default storage class running. If not, [set up a default storage class on Google Kubernetes Engine](https://cloud.google.com/anthos/clusters/docs/on-prem/1.3/how-to/default-storage-class)[GKE].
+  3. The NGINX Ingress Controller should be enabled by default. If not, [install ingress controller on Google Kubernetes Engine](https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke)[GKE]. Ensure that you enable the correct version for your cluster.
 
+</TabItem>
+<TabItem label="AWS EKS" value="AWSEKS">
+
+   1. Set up the Kubernetes cluster by [creating a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+    2. Ensure that you have a default storage class running. If not, [create default storage class](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
+    3. The NGINX Ingress Controller should be enabled by default. If not, [install NGINX Controller for AWS EKS](https://kubernetes.github.io/ingress-nginx/deploy/#network-load-balancer-nlb).
+    Ensure that you enable the correct version for your cluster.
+
+</TabItem>
+<TabItem label="Microk8s" value="Microk8s">
+
+  1. Set up Kubernetes by following [working with kubectl](https://microk8s.io/docs/working-with-kubectl).
+  2. Ensure that you have a default storage class running. If not, [enable storage class](https://microk8s.io/docs/command-reference#heading--microk8s-enable).
+  3. The NGINX Ingress Controller should be enabled by default. If not, [add on: Ingress](https://microk8s.io/docs/addon-ingress). Ensure that you enable the correct version for your cluster.
+    
+</TabItem>
+</Tabs>
+
+## Install
+
+1. Click to download the [`values.yaml`](https://bit.ly/3ETEgPT) file, and place it into the Appsmith installation folder.
```

Comment:
Okay.

Reply:
> > @danciaclara - The markdown table do not have a fixed width and driven by content that is displayed. Because of which looks weird with varied widths on same page. I think, we can add some style changes to the same. Your thoughts?
> 
> @jnikhila Hey didn't understand the context here. Can you share some screenshots?

As discussed - I can look into having a custom CSS for markdown tables. However, that's not a blocking change for this PR.